packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -245,18 +245,19 @@ function* watchEditTrack() {
 function* deleteTrackAsync(
   action: ReturnType<typeof trackActions.deleteTrack>
 ) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   yield* waitForWrite()
-  const userId = yield* select(getUserId)
```

Comment:
Why this change? Seems unrelated to socials.

Reply:
because we need the user to get the `artist_pick_track_id`. see line 260 of new code